snowpenguin/django/recaptcha2/templates/snowpenguin/recaptcha/recaptcha_explicit.html

```diff
@@ -1,5 +1,6 @@
-<div id="{{ container_id }}"></div>
+<div id="{{ container_id }}" class="g-recaptcha"></div>
 <script>
+    var django_recaptcha_callbacks = django_recaptcha_callbacks || [];
```

Comment:
Rather than add a script on every instance of RRRRRReeeeecapppptchhhaaa, could we defer this to the end 

```
const id = document.querySelectorAll('.g-recaptach').map(el => {
const containerID = el.id
  grecaptcha.render(containerID, { ... all that junk ...})
  // Maybe call the function now ??
})